### PR TITLE
confusing overriding of variables

### PR DESCRIPTION
**Pull request by pkellner**
I see this a lot and I always think it's a bad practice. Not sure "b" is the right choice,but "builder" is definitely one of the few wrong choices.

### CODE REVIEW

**Comment by serpent5** [comment]
We don't use single-letter names in the docs, so I suggest `policy` as something that's consistent with e.g. [the authz policies article](https://docs.microsoft.com/en-us/aspnet/core/security/authorization/policies?view=aspnetcore-6.0).

**Review by serpent5 — COMMENTED** [review]
We don't use single-letter names in the docs, so I suggest `policy` as something that's consistent with e.g. [the authz policies article](https://docs.microsoft.com/en-us/aspnet/core/security/authorization/policies?view=aspnetcore-6.0).

File: aspnetcore/security/cors/6.0sample/Cors/WebAPI/Program.cs [hunk 1]
```diff
@@ -9,9 +9,9 @@
 builder.Services.AddCors(options =>
 {
     options.AddPolicy(name: MyAllowSpecificOrigins,
-                      builder =>
+                      b =>
```
Comment (Rick-Anderson) [inline aspnetcore/security/cors/6.0sample/Cors/WebAPI/Program.cs:12]:
```suggestion
                      policy  =>
```

File: aspnetcore/security/cors/6.0sample/Cors/WebAPI/Program.cs [hunk 2]
```diff
@@ -9,9 +9,9 @@
...
                       {
-                          builder.WithOrigins("http://example.com",
+                          b.WithOrigins("http://example.com",
```
Comment (Rick-Anderson) [inline aspnetcore/security/cors/6.0sample/Cors/WebAPI/Program.cs:14]:
```suggestion
                          policy.WithOrigins("http://example.com",
```

**Review by Rick-Anderson — CHANGES_REQUESTED** [review]

**Review by serpent5 — APPROVED** [review]

**Review by Rick-Anderson — APPROVED** [review]

**Comment by pkellner** [comment]
Wow!  Thank you for fixing this.

**Comment by Rick-Anderson** [comment]
@pkellner  thanks for pointing it out. You get the commit credit.

**Pull request merged**